src/test/unit/test_lumigo_utils.py

```diff
@@ -400,6 +400,13 @@ def test_config_step_function_without_envs(monkeypatch, configuration_value):
     assert Configuration.is_step_function == configuration_value
 
 
+@pytest.mark.parametrize("value, expected", (("TRUE", True), ("FALSE", False)))
+def test_config_propagate_w3c_by_env(monkeypatch, value, expected):
+    monkeypatch.setenv("LUMIGO_PROPAGATE_W3C", value)
```

Comment:
Can you add another test for the default value? Where the env var "LUMIGO_PROPAGATE_W3C" isn't set.